docs/src/apps/jsonrpc-api.md

```diff
@@ -213,6 +214,32 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 {"jsonrpc":"2.0","result":{"commitment":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,10,32],"totalStake": 42},"id":1}
 ```
 
+### getBlockhashQueueLength
+
+Returns the blockhash queue length for the current epoch.
+
+#### Parameters:
+
+None
+
+#### Results:
+
+The result will be an RpcResponse JSON object with `value` equal to and object
+with the following fields:
+
+* `epoch: <u64>` - epoch
+* `blockhashQueueLength: <usize>` - length of the blockhash queue
```

Comment:
There could be more guidance here on what assumptions can be made based off the value of `blockhashQueueLength`.    I think it's: you can assume your blockhash is valid for N slots, where N = "slot that the blockhash you have is from" + blockhashQueueLength.    Is that right?

Reply:
If so, I wonder if `blockhashQueueLength` would be better named something like `blockhashTTL` or `blockhashLifespan`

Reply:
> I think it's: you can assume your blockhash is valid for N slots, where N = "slot that the blockhash you have is from" + blockhashQueueLength. Is that right?

Yes, exactly!
In the interest of not adding initializations (even though I know its a standard one), I like `blockhashLifespan`.